src/views/MorphicBarEditor.vue

```diff
@@ -495,11 +491,11 @@ export default {
       return buttons
     },
     getPredefinedButtons: function () {
-      let buttons = []
+      const buttons = []
```

Comment:
Same here, empty constant that you are starting modifying right away, makes no sense.